src/components/settings/SettingsHeader/SettingsHeader.scss

```diff
@@ -0,0 +1,27 @@
+.SettingsHeader {
+  margin-top: 0.5rem;
+  margin-bottom: 1.25rem;
+
+  @media only screen and (max-width: 700px) {
+    display: none;
+  }
```

Comment:
Shouldn't we go with a mobile-first approach throughout the entire codebase? So, using `min-width` instead of `max-width` in media

Reply:
I think for short time this is not worth it. It will take a lot of time and we don't get any output results (UI stays the same). But I agree that this approach is the way to go, let's revisit this when we do the UI refactor.